### PR TITLE
Ct/finicity fix

### PR DESCRIPTION
**Pull request by Craiting**
Finicity doesn't have have a way to create connections via an api so you have to create a customer via the api and then generate a widget_url with that customer. So in our app we're using the Oauth flow and webhooks to know when the connection has been made in the other window. The widget will successfully create a connection with Finicity but it's currently not returning any data properly, and Finicity probably doesn't have any VC endpoints to get the data from. This PR is just to get the connection working and write a test similar to the MX OAuth test.

The team should discuss what to do about getting Finicity data working or even if we want to support that in the MVP.

### CODE REVIEW

File: server/providers/finicity.ts [hunk 1]
```diff
@@ -31,12 +27,12 @@ export class FinicityApi implements ProviderApiClient {
   }
 
   async GetInstitutionById (id: string): Promise<Institution> {
-    const ins = this.apiClient.getInstitution(id)
+    const institution = await this.apiClient.getInstitution(id)
```
Comment (wesrisenmay-mx) [inline server/providers/finicity.ts:30]: At a minimum we should unit test the functions we touched in this file

File: cypress/e2e/finicity.cy.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  beforeEach(() => {
+    cy.setAuthCode()
+  })
+
+  it('Connects to Finbank profiles a', () => {
+    const authCode = Cypress.env('authCode')
+
+    cy.visit(`http://localhost:8080/?job_type=agg&auth=${authCode}`)
+    cy.findByPlaceholderText('Search').type('finbank profiles')
+    cy.findByLabelText('Add account with finbank profiles - a').first().click()
```
Comment (wesrisenmay-mx) [inline cypress/e2e/finicity.cy.ts:11]: Do you need the .first()?

**Review by wesrisenmay-mx — APPROVED** [review]
Nice work.

File: cypress/e2e/finicity.cy.ts (hunk 2 above)
Comment (Craiting) [reply cypress/e2e/finicity.cy.ts]: It turns out that I don't. Removed

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**